pkg/feature/builder.go

```diff
@@ -220,3 +222,9 @@ func (fb *featureBuilder) TargetNamespace(targetNs string) *featureBuilder {
 
 	return fb
 }
+
+func (fb *featureBuilder) Managed() *featureBuilder {
```

Comment:
i dont have objection for this PR, 
but just my thoughts, why not have this as 
```
func (fb *featureBuilder) Managed(isManaged bool) *featureBuilder {
  fb.managed = isManaged
```
then in `markAsManaged()` you always set annotation but with true of false as value (in this case it probably wont be called as markAsManaged any more)

Reply:
The thinking there was because if the annotation is not present, then we do not manage. So the utility in setting all resources explicitly as non-managed was minimal, if they would already not have the annotation present.